File: pra.pl
#/usr/bin/perl
use strict;
use warnings;

my $n = <STDIN>;
my @a = split ' ', <STDIN>;
my $ans = 0;
for my $i (0..$n-1) {
    for my $j ($i+1..$n-1) {
        $ans++ if $a[$i] > $a[$j];
    }
}
print "$ans\n";

File: pra.rb
#/usr/bin/ruby
n = gets.to_i
a = gets.split.map(&:to_i)
ans = 0
(0...n).each do |i|
    (i+1...n).each do |j|
        ans += 1 if a[i] > a[j]
    end
end
puts ans